src/Apps/CollectorProfile/Routes/Saves2/Components/SavesHeader.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { useState } from "react"
 import { Box, Text, Spacer, Button, Join } from "@artsy/palette"
-import { CreateNewListModalContainer } from "./CreateNewListModal"
 import { useToasts } from "@artsy/palette"
 import { useTranslation } from "react-i18next"
+import { CreateNewListModalWizard } from "Apps/CollectorProfile/Routes/Saves2/Components/CreateNewList/CreateNewListModalWizard"
```

Comment:
Going to move this file up one directory (all create list-related files will be located in `CreateNewList` directory). I was waiting until your other PR got merged @dimatretyak